hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/TestRMFailover.java

```diff
@@ -212,6 +213,9 @@ public void testAutomaticFailover()
   public void testWebAppProxyInStandAloneMode() throws YarnException,
       InterruptedException, IOException {
     conf.setBoolean(YarnConfiguration.AUTO_FAILOVER_ENABLED, false);
+    if (HAUtil.isHAEnabled(conf)) {
```

Comment:
`HAUtil.isHAEnabled()` is always true, since it is set in the setup of this test here: https://github.com/apache/hadoop/blob/trunk/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/TestRMFailover.java#L88

Can you remove the `if` condition?

Reply:
Thanks for your suggestion! I will remove `if (HAUtil.isHAEnabled(conf))`.